src/test/ui/issue-23302-1.rs

```diff
@@ -11,7 +11,7 @@
 // Check that an enum with recursion in the discriminant throws
 // the appropriate error (rather than, say, blowing the stack).
 enum X {
-    A = X::A as isize, //~ ERROR E0391
+    A = X::A as isize, //~ ERROR enums without inhabited variants do not have discriminants
```

Comment:
This makes the test no longer be a test of what it used to be testing. Could you add another variant `B` before `A`?